tss-esapi/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tss-esapi"
-version = "4.0.10-alpha.2"
+version = "4.0.11-alpha.1"
```

Comment:
```suggestion
version = "4.0.10-alpha.3"
```

Reply:
is that for breaking changes? Not just bug fixes?

Reply:
Well moving stuff around breaks others code. 

I must say it is slightly odd that we increase minor version on something that is yet to be released. Would make more sense to just 4.0.0-alpha.x all the way until we are happy with the alpha.

Reply:
Yeah, we should've been increasing the alpha version from the beginning 😓

Reply:
As I said on the channel, would it be that bad to release a `5.0.0`?? If we are not making releases that often now. Is there a big difference if our real "stable" is `54.0.0` instead of `5.0.0`?